helpers/handlebars/middleware.js

```diff
@@ -179,7 +179,7 @@ module.exports = (req, res, next) => {
 
 	// admin views
 	res.locals.sidebarItems.push({
-		name: res.$t('global.sidebar.link.administration'),
```

Comment:
Is the varibale still in use somewhere? If not, please delete it from the language file

Reply:
deleted ;)